x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/rules/action/DeleteQueryRulesetAction.java

```diff
@@ -29,9 +35,11 @@ private DeleteQueryRulesetAction() {
         super(NAME, AcknowledgedResponse::readFrom);
     }
 
-    public static class Request extends ActionRequest {
+    public static class Request extends ActionRequest implements ToXContentObject {
         private final String rulesetId;
 
+        public static final ParseField RULES_ID = new ParseField("rules_id");
```

Comment:
Nitpick: Can we please rename this to `RULESET_ID_FIELD`? Also I think we want the value to be `ruleset_id` not `rules_id` to be consistent with our [docs](https://github.com/elastic/elasticsearch/blob/main/docs/reference/query-rules/apis/delete-query-ruleset.asciidoc)?